CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 25.02+ (???)
 ------------------------------------------------------------------------
+- Feature: [#3047] Add RE 4/4 OpenGraphics vehicle, add OpenGraphics interface skin, add all replacement OpenGraphics walls.
```

Comment:
I would just write "Add OpenGraphics replacements for RE 4/4, the interface skin, and all walls."